src/renderer-backend-egl.cpp

```diff
@@ -139,6 +139,11 @@ class Target {
 
     ~Target()
     {
+        if (m_wl.wpeBridgeId && m_glib.socket) {
+            uint32_t message[] = { 0x43, m_wl.wpeBridgeId };
```

Comment:
Ouch... using “magic” numbers does not make me super happy.

Reply:
I agree, existing code uses 0x42 for the other message so I just followed what we have.